package.json

```diff
@@ -22,8 +22,7 @@
     "format:fix:prettier": "prettier --write --cache .",
     "e2e": "cypress open",
     "build:packages": "npm run build -w=packages/remix-runtime -w=packages/remix-adapter -w=packages/remix-edge-adapter",
-    "build:packages:watch": "npm run build:watch -w=packages/remix-runtime & npm run build:watch -w=packages/remix-edge-adapter",
-    "build:demo": "npm run build -w=packages/demo-site"
```

Comment:
This script it no longer required with the new monorepo support. We can run `npm run build -w=packages/demo-site` directly as the `cwd` is the monorepo root.